tests/processor/test_utils.py

```diff
@@ -155,18 +127,141 @@ def test_unpack_rpm(m_rpm_to_cpio, m_unpack_cpio, m_assert_command):
     m_unpack_cpio.assert_called_once_with(cpio, output_dir)
 
 
-@mock.patch('os.mkdir')
+class TestContainerUnpacking:
+    """Container unpacking test with enviroment setup."""
+
+    @staticmethod
+    def _create_container_image(temp_dir):
```

Comment:
Optional: This is a nice elegant way of doing this but since you only use this once, perhaps you'd consider just fabricating this tar file and commit to the tests. That way it's less test code we need to maintain. I'll leave it up to you. It's fine to keep it as is.